packages/gatsby/src/utils/__tests__/websocket-manager.ts

```diff
@@ -500,11 +500,9 @@ describe(`websocket-manager`, () => {
 
   describe(`Data`, () => {
     beforeEach(() => {
-      websocketManager.pageResults.clear()
       websocketManager.staticQueryResults.clear()
     })
     afterAll(() => {
-      websocketManager.pageResults.clear()
```

Comment:
`pageResults` were never used for anything (probably after some of the refactors) and were just never removed before. I can revert those changes as this is not strictly related to slices (and maybe open seperate PR for it)